setup.sh

```diff
@@ -13,7 +13,7 @@ cp ./user-utils/src/df/test.py $HOME/.deepnote
 cp ./user-utils/src/variable_explorer/variable_explorer.py $HOME/.deepnote
 cp ./user-utils/src/variable_explorer/variable_explorer_helpers.py $HOME/.deepnote
 
-
-
+# Get available kernels
+jupyter kernelspec list --json | jq -r '.kernelspecs | keys[]' > $HOME/work/.deepnote/available-kernels.txt
```

Comment:
I would keep the whole json, there's stuff like `display_name` which we could use for the UI elements:

```suggestion
jupyter kernelspec list --json > $HOME/work/.deepnote/available-kernels.json
```

```
{
  "kernelspecs": {
    "python3": {
      "resource_dir": "/opt/venv/share/jupyter/kernels/python3",
      "spec": {
        "argv": [
          "python",
          "-m",
          "ipykernel_launcher",
          "-f",
          "{connection_file}"
        ],
        "env": {},
        "display_name": "Python 3",
        "language": "python",
        "interrupt_mode": "signal",
        "metadata": {}
      }
    }
  }
}
```

Reply:
why no ok